test/js/node/stream/node-stream.test.js

```diff
@@ -386,6 +386,27 @@ it("Readable.fromWeb", async () => {
   expect(Buffer.concat(chunks).toString()).toBe("Hello World!\n");
 });
 
+it("Writable.toWeb", async () => {
+  let dataBuffer = "";
+
+  // Create a Node.js Writable stream
+  const nodeWritable = new Writable({
+    write(chunk, encoding, callback) {
+      dataBuffer += chunk.toString();
+      callback();
+    },
+  });
+
+  const webWritable = Writable.toWeb(nodeWritable);
+  const writer = webWritable.getWriter();
+
+  // Write some data
+  const encoder = new TextEncoder();
+  await writer.write(encoder.encode("Hello, World!"));
```

Comment:
This should have tests for multiple chunks, not just one.

Reply:
modified.